### PR TITLE
Fixes optional price behavior

### PR DESCRIPTION
**Pull request by lightsprint09**
As described in fptf amount and currency should be required. If amount is not available the whole price object is nil

```
price: { // optional
	amount: 12.50, // number, required
	currency: 'EUR' // ISO 4217 code, required
}
```

### CODE REVIEW

File: p/db/index.js [hunk 1]
```diff
@@ -55,20 +55,18 @@ const createParseJourney = (profile, opt, data) => {
 		// 	targetCtx: 'D',
 		// 	buttonText: 'To offer selection'
 		// } ]
-		res.price = {amount: null, hint: 'No pricing information available.'}
```
Comment (derhuerst) [inline p/db/index.js]: Please assign `null` as the default value.

Comment (lightsprint09) [reply p/db/index.js]: Added `null` as default

**Comment by derhuerst** [comment]
Technically someone could rely one the behaviour that, if `journey.price` is truthy, it will contain a price. Because of this, I'm not sure if this is a breaking change.

**Comment by derhuerst** [comment]
Sorry, I just noticed that I have added this behaviour in 8677a1cc on branch `5` already. Nonetheless, thanks for your contribution!

**Pull request closed without merging**